group_vars/mainnet.yml

```diff
@@ -5,3 +5,9 @@ chainflip:
     version: 1.7.5
   engine:
     version: 1.7.5
+    shared_libraries:
+    - libchainflip_engine_v1_6_8.so
+    - libchainflip_engine_v1_7_5.so
+  promtail:
+    node_id:
+    node_vanity_name:
```

Comment:
Please add the values as well https://www.notion.so/encapsulate/testnet-14714f565a5080109c92e6db7d4bb8bf?pvs=4#14c14f565a5080528a3aeea2a790b127